apps/organizations/models.py

```diff
@@ -320,18 +321,48 @@ def status(self) -> str:
     ##
     # Notifications
 
+    User = get_user_model()
+
+    def _build_notification_message(self, subject: User, verb: str) -> str:
```

Comment:
For now, HTK has to be Python2 compatible (there are many users of django-htk still on Python 2, sadly), so please remove all of the type-hints as shown in the suggested code

```suggestion
    def _build_notification_message(self, subject, verb):
```